app/lib/finder_api.rb

```diff
@@ -129,6 +129,7 @@ def query_builder_class
 
   FINDERS_IN_DEVELOPMENT = {
     "/news-and-communications" => "news_and_communications",
+    "/guidance_and_regulation" => "guidance_and_regulation",
```

Comment:
The other supergroup finders don't have underscores in the URL.

Reply:
Done